src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -2190,4 +2202,15 @@ PeerImp::getScore (bool haveItem)
    return score;
 }
 
+bool
+PeerImp::highLatency ()
```

Comment:
`getRemoteAddress` was written with const-correctness in mind. Having gotten more experience with it, I conclude that it did not offer much if anything. No bug was fixed and no problem avoided because `const` appeared on a virtual member. I'm not against making the members const when they could be,  but `const` aint exactly walking on water here either.